docs/site/HasMany-relation.md

```diff
@@ -314,3 +314,116 @@ certain properties from the JSON/OpenAPI spec schema built for the `requestBody`
 payload. See its [GitHub
 issue](https://github.com/strongloop/loopback-next/issues/1179) to follow the discussion.
 " %}
+
+## Querying related models
+
+Loopback4 has the concept of `inclusion resolver` in relations, which helps to
+query data through include filter. An inclusion resolver is a function that can
+fetch target models for the given list of source model instances. LB4 creates a
+different inclusion resolver for each relation type.
+
+The following is an example for has-many inclusion resolver:
+
+- Model `Customer`
+- Model `Order`
+- `Customer` has mamy `Order`
+
+```ts
+// import statements
+class Customer extends Entity {
+  // property definition for id, name
+  @hasMany(() => Order, {keyTo: 'customerId'})
+  orders?: Order[];
+}
+```
+
+```ts
+// import statements
+class Order extends Entity {
+  // property definition for id, name
+  @property({
+    type: 'number',
+  })
+  customerId?: number;
+  // constructor, relation, etc
+}
+```
+
+The inclusion resolver allows users to retrieve all customers along with all
+related orders by the following code:
+
+```ts
+categoryRepo.find({include: [{relation: 'products'}]});
```

Comment:
Right, I forgot to change it to `customerRepo`, good catch!

In that PR, she modifies the way to get resolvers so that we don't have to override the find method.

Reply:
Right, I forgot to change it to customerRepo, good catch!

In that PR, she modifies the way to get resolvers so that we don't have to override the find method.